src/utils/generation_utils.py

```diff
@@ -343,15 +402,21 @@ def do_generation(
         chat_history_file=context.chat_history.file, filter_for=generation_for
     )
 
+    if prompt_block.index_in_file not in block_indices:
+        block_indices.append(prompt_block.index_in_file)
+
     options = {}
     if stop_tokens:
         options["stop"] = stop_tokens
 
     output_file_id = None if new_file else context.chat_history.file.id
 
+    # don't pollute workspace with temporary/working files that contain data like: "LIKELY"
+    append_output_to_file = False if not output_file_id else True
+
     task = generator.generate(
         tags=output_tags,
-        append_output_to_file=True,
+        append_output_to_file=append_output_to_file,
```

Comment:
My logic for having these go into a different File in the workspace, as opposed to being purely ephemeral, was mostly that you could recreate everything that had happened by looking at the content of the workspace.  I'm not opposed to doing this, though.